portfolio/src/main/webapp/index.html

```diff
@@ -263,27 +263,27 @@ <h4 class="indigo-text text-darken-t4">
 
   <div class="row">
       <h5 >Feel free to contact me.&nbsp;&nbsp;</h5>
-    <form action="get-comment" class="col s12">
+    <form action="/data" class="col s12">
```

Comment:
should this action be "/new-post"?

Reply:
My bad. I forgot to change after changing the servlet name on the .java file.